docs/contributing/guidelines-for-writing-documentation.md

```diff
@@ -10,6 +10,7 @@
     * The main [`mkdocs.yml`](https://github.com/shopsys/shopsys/blob/7.3/mkdocs.yml) configuration file is located in the monorepo root.
 * All documentation files should be named same as first heading (in lowercase, non-alphanumeric characters replaced by dash).
 * References to project files and classes have to be absolute links to the GitHub in proper version - e.g. `[app/config/config.yml](https://github.com/shopsys/shopsys/blob/7.3/project-base/app/config/config.yml)`.
+    * there is an exception for files [CHANGELOG](https://github.com/shopsys/shopsys/blob/master/UPGRADE.md), [UPGRADE](https://github.com/shopsys/shopsys/blob/master/UPGRADE.md), and [Open Source License Acknowledgements and Third-Party Copyrights](https://github.com/shopsys/shopsys/blob/master/open-source-license-acknowledgements-and-third-party-copyrights.md) - these files should be always linked in the latest (i.e. `master`) version.
```

Comment:
are you sure? imho the `Open Source License Acknowledgements and Third-Party Copyrights` should link to respective version because the library might be removed in newer version...

Reply:
Oh, we were sure about that, but this is a fair point :stuck_out_tongue:

Reply:
I have removed the note from the docs and created a new PR fo fixing the links - https://github.com/shopsys/shopsys/pull/1466